webapp/components/admin_console/select_team_modal.jsx

```diff
@@ -29,20 +29,18 @@ export default class SelectTeamModal extends React.Component {
         }
 
         var options = [];
-
-        for (var key in this.props.teams) {
-            if (this.props.teams.hasOwnProperty(key)) {
-                var team = this.props.teams[key];
-                options.push(
-                    <option
-                        key={'opt_' + team.id}
-                        value={team.id}
-                    >
-                        {team.name}
-                    </option>
-                );
-            }
-        }
+        var teams = this.props.teams;
+        Reflect.ownKeys(teams).forEach((key) => {
+            var team = teams[key];
+            options.push(
+                <option
+                    key={'opt_' + team.id}
+                    value={team.id}
+                >
+                    {team.display_name}
```

Comment:
We know that 'for...in loop' iterates over all enumerable properties of an object, including it's enumerable prototype properties. This traversing of prototype chains makes drawback in performance. Reflect.ownKeys() and Object.getOwnPropertyNames() prevent this drawback. Also from our test, Array.prototype's forEach shows best loop performance in modern browsers.